x-pack/plugins/observability_ai_assistant/server/functions/kibana.ts

```diff
@@ -76,6 +78,9 @@ export function registerKibanaFunction({
         url: format(nextUrl),
         data: body ? JSON.stringify(body) : undefined,
         signal,
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false,
+        }),
```

Comment:
I'm assuming we _think_ this is safe because the request is coming from within Kibana to Kibana itself, right?

Reply:
Yes, but I'm mostly just testing still. Should have mentioned that.